Tests/iOS/Tests/Storage/HybridStorageTests.swift

```diff
@@ -0,0 +1,139 @@
+import XCTest
+import SwiftHash
+@testable import Cache
+
+final class HybridStorageTests: XCTestCase {
+  private let cacheName = "WeirdoCache"
+  private let key = "alongweirdkey"
+  private let testObject = User(firstName: "John", lastName: "Targaryen")
+  private var storage: HybridStorage!
+  private let fileManager = FileManager()
+
+  override func setUp() {
+    super.setUp()
+    let memory = MemoryStorage(config: MemoryConfig())
+    let disk = try! DiskStorage(config: DiskConfig(name: "Floppy"))
+
+    storage = HybridStorage(memoryStorage: memory, diskStorage: disk)
+  }
+
+  override func tearDown() {
+    try? storage.removeAll()
+    super.tearDown()
+  }
+
+  func testSetObject() throws {
+    try when("set to storage") {
+      try storage.setObject(testObject, forKey: key)
```

Comment:
Don't you need to use `self` in closure? 🤔

Reply:
@vadymmarkov closure has `no escaping` by default in Swift 3+ ?